site/content/in-dev/unreleased/overview.md

```diff
@@ -68,16 +68,6 @@ A catalog is configured with a storage configuration that can point to S3, Azure
 You create *namespaces* to logically group Iceberg tables within a catalog. A catalog can have multiple namespaces. You can also create
 nested namespaces. Iceberg tables belong to namespaces.
 
-### Apache Iceberg&trade; tables and catalogs
-
-In an internal catalog, an Iceberg table is registered in Polaris, but read and written via query engines. The table data and
-metadata is stored in your external cloud storage. The table uses Polaris as the Iceberg catalog.
-
-If you have tables housed in another Iceberg catalog, you can sync these tables to an external catalog in Polaris.
-If you sync this catalog to Polaris, it appears as an external catalog in Polaris. Clients connecting to the external
-catalog can read from or write to these tables. However, clients connecting to Polaris will only be able to
-read from these tables.
-
```

Comment:
We can remove this part, given the above catalog section already describes internal and external catalog.